ci/ci/ci.py

```diff
@@ -20,9 +20,7 @@
     Database,
     check_csrf_token,
     monitor_endpoints_middleware,
-    rest_authenticated_developers_only,
     setup_aiohttp_session,
-    web_authenticated_developers_only,
```

Comment:
Was this removal intentional (not included in the PR title)?

Reply:
Yes, updated the title :)